client/src/components/HistoryViewer/HistoryViewer.js

```diff
@@ -65,6 +66,22 @@ class HistoryViewer extends Component {
     return edges.map((version) => version.node);
   }
 
+  /**
+   * Returns a string to be used as the "class" attribute on the history viewer container
+   *
+   * @returns {string}
+   */
+  getContainerClasses() {
+    const classes = ['history-viewer', 'fill-height'];
+
+    if (this.props.compare) {
+      classes.push('compare-mode');
```

Comment:
can you make this `history-viewer--compare-mode` please?

Reply:
Done 🙂

Reply:
Wait, it should be `history-viewer__compare-mode` - one moment please

Reply:
Ok actually done 🙂